### ISSUE TITLE
Unsafe assignment/member access/call of an `any` value.

### ISSUE THREAD

**Issue by jpnws** [UrielCh/adbkit#8]
@UrielCh I was having trouble with @devicefarmer/adbkit recently when I tried using `device.openLogcat`. Specifically, the problem with that seemed to have been for some reason the `Logcat` type information wasn't being detected. For instance, if you take a look at the following code.
![2022-09-09-13 05 08@2x](https://user-images.githubusercontent.com/61026408/189263732-e3ea049e-ea64-42b9-95c0-a6c0b1526838.png)
You will notice that vscode does not give me any suggestions on the `Logcat` class implementation. I tried installing @devicefarmer/adbkit-logcat separately, but decided not to use it separately becuase it appeared as if @devicefarmer/adbkit library already had adbkit-logcat integrated. Anyway, I didn't want to dig any deeper into this issue with @devicefarmer and that's when I found your fork. Given that your version uses pure Typescript, I uninstalled bluebird and all the @devicefarmer packages and tried using your version.

I hoped it would have been pretty much plug and play, replace and move on, but it seems like there are a little more things I need to do before using your version properly.

So, basically, I imported your adbkit, and updated the code to execute `createClient` function directly, instead of doing something like `Adb.createClient()` as it was with @devicefarmer. After doing this, I started getting some eslint errors relating to unsafe assignment/member access/call of an `any` value.

![2022-09-09-13 10 41@2x](https://user-images.githubusercontent.com/61026408/189264504-4675bbf9-3e2c-4da6-930f-76918b828950.png)

I wasn't sure whether this was directly related to the way your adbkit pure Typescript version was written or if it had something to do with my usage of the library, but I thought I'd consult with you to check if you had any thoughts on it.

**Comment by UrielCh** [comment]
please include your include statements.

**Comment by jpnws** [comment]
Thanks for the response, I've actually moved on with different approach for now and I don't have the code with adbkit anymore.
